### ISSUE TITLE
number of epochs

### ISSUE THREAD

**Issue by qwenzo** [SakanaAI/text-to-lora#11]
Hello,

Thank you for the great work! 
In the paper, you mentioned that you train t2l for 100k epochs but I see in the training script you have only 20k epochs so which one is used to generate the results? 
Also is there a reason for choosing such a high nubmer of epochs? I ran training and it seems like the loss converages afer a few epochs only.
